### ISSUE TITLE
Generate NonStandard InChI

### ISSUE THREAD

**Issue by johnmay** [cdk/cdk#253]
Via Nina:

```java
Using InChIGenerator with non-standard options generates standard InChI (starting with 1S/ ) with 1.5.14. The same code generated non-standard InChI in previous versions (1.5.13 and before)

List<INCHI_OPTION> options = new ArrayList<INCHI_OPTION>();
options.add(INCHI_OPTION.FixedH);
options.add(INCHI_OPTION.SAbs);
options.add(INCHI_OPTION.SAsXYZ);
options.add(INCHI_OPTION.SPXYZ);
options.add(INCHI_OPTION.FixSp3Bug);
options.add(INCHI_OPTION.AuxNone);
InChIGeneratorFactory factory = InChIGeneratorFactory.getInstance();
InChIGenerator gen1 = factory.getInChIGenerator(mol1, options);
```

**Comment by johnmay** [comment]
@vedina have you been able to confirm if this is a dependency problem?

**Comment by vedina** [comment]
it certainly affects cdk 1.5.14 and not cdk 1.5.13. 

had not checked if the underlying inchi library is different - is it?

**Comment by johnmay** [comment]
No it's not - but there are lots of test cases that generate non-standard InChIs thats didn't fail. Can you check on 2.0-SNAPSHOT.

John

**Comment by johnmay** [comment]
Oh and I added you exact test case to check - the output is correct:

https://jenkins.bigcat.unimaas.nl/job/cdk/lastCompletedBuild/testReport/org.openscience.cdk.inchi/InChIGeneratorTest/nonStandardInChIWithEnumOptions/

https://github.com/cdk/cdk/blob/master/storage/inchi/src/test/java/org/openscience/cdk/inchi/InChIGeneratorTest.java#L230

**Comment by johnmay** [comment]
@vedina will close 1 day please let me know if you want to keep open

**Comment by vedina** [comment]
will check during the weekend, there was a problem when upgrading to 1.5.14

**Comment by vedina** [comment]
@johnmay here is the test 

https://github.com/ideaconsult/examples-cdk/blob/master/maven-single-module/src/test/java/net/idea/examples/cdk/maven_single_module/InchiTest.java

Works with cdk <=1.5.13, fails with 1.5.14 . There is maven profile for CDK versions .

**Comment by vedina** [comment]
@johnmay it's weird, the tests (including the one in CDK) fail only on Windows machines (both JDK 7 and 8). Any hints what might have changed since 1.5.13 will be very useful!

**Comment by johnmay** [comment]
Ahhhh that makes more sense. Putting this together with something I saw on the InChI mailing list. InChI it's it's infinite wisdom as different syntax depending on the OS. Windows uses '/' and everything else uses '-'.

Windows:
```inchi-1 /STDIO```
Everything Else:
```inchi-1 -STDIO```

When we modified the handling to allow the 15T/KET etc I added output as '-KET -15T' etc. I really thought the '/' was needed at the API level... but apparently this is the case.

I'll get this fixed ASAP, hopefully by tomorrow. 

Here (https://github.com/cdk/cdk/blob/master/storage/inchi/src/main/java/org/openscience/cdk/inchi/JniInChIInputAdapter.java#L79) the '-' needs to be replaced with a constant based on OS (like line separation).
